guide/C/ch_invest.xml

```diff
@@ -971,56 +980,165 @@ Income
+            price not adjusted by commission and enter the commission expense on
+            a separate split in the transaction. This enables the tracking of
+            commission but is <emphasis>not</emphasis> compatible with using
+            <xref linkend="invest-sellLots"></xref>. Scrubbing doesn't know to
+            deduct commissions and fees from the gains, so capital gains or
+            losses must be manually calculated (see
+            <xref linkend="invest-sell-man"></xref>).</para>
+          </listitem>
+        </itemizedlist>
+      </sect3>
+
+      <sect3 id="invest-buy-gross">
+        <title>Example: Buying Shares with Gross Pricing</title>
+
+        <para>In this example you will purchase $5000 of IBM stock, with a
+          commission of $100. First step will be to create the stock account for
+          IBM. The existing <emphasis>Expenses:Commissions</emphasis>
+          account will be used. If you wish to track commissions to the
+          individual stock level an additional sub-account would be necessary.
+          E.g. <emphasis>Expenses:Commissions:IBM</emphasis>.
+        </para>
 
+        <para>Now for the transaction, on the first (transaction) line, enter
+          the <guilabel>Date</guilabel> of the purchase (eg: Jan. 3 2005) and
+          <guilabel>Description</guilabel> (eg: Initial IBM Purchase). On the
+          first split line, <guilabel>Shares</guilabel> will be skipped (to be
+          calculated), enter <guilabel>Price</guilabel> (eg: $96.6001), and
+          <guilabel>Buy</guilabel> ($5000). You do not need to fill in the
+          <guilabel>Shares</guilabel> column, as it will be calculated for you
+          when you <keycap>Tab</keycap> to the next split.
+          The next line in the split transaction will be <emphasis>
+          Expenses:Commissions</emphasis> and fill in <guilabel>Buy</guilabel>
+          ($100). The third line will be to transfer $5100 from account <emphasis>
+          Assets:Bank ABC</emphasis> to balance the transaction. Your IBM
+          Commodity view should now appear like this:</para>
+
+        <screenshot id="invest-setup-portfolio2">
+          <mediaobject>
+             <imageobject role="html">
+               <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
+                          srccredit="Cristian Marchi" width="510px"></imagedata>
+             </imageobject>
+             <imageobject role="fo">
+               <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
+                          srccredit="Cristian Marchi"></imagedata>
+             </imageobject>
+
+            <textobject>
+              <phrase>Setup Current Portfolio</phrase>
+            </textobject>
+
+            <caption><para>This image is of the transaction register of the IBM account
+            after the first stock account <quote>purchase</quote> with a commission.</para>
```

Comment:
I would always enter Shares and Buy and let the program calculate the Prize. I paid an amount of money and got a number of shares.
Any other combination can result in rounding errors.

Probably we should add an explaining note already in 
9.5.1. Entering Preexisting Shares.